spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/HyperConformance.scala

```diff
@@ -65,7 +65,7 @@ class HyperConformance (implicit cmd: ConformanceConfig,
     import za.co.absa.enceladus.utils.implicits.DataFrameImplicits.DataFrameEnhancements
 
     val infoDateColumn = infoDateFactory.getInfoDateColumn(rawDf)
-    val infoVersionColumn = infoVersionFactory.getInfoVersionColumn(rawDf)
+    val infoVersionCol = infoVersionFactory.getInfoVersionColumn(rawDf)
```

Comment:
I think you changed at wrong place. Here it was consistent, the inconsistency was on line 146 😉